x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/asset_criticality/host_flyout.cy.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../../objects/rule';
+import {
+  HOST_DETAILS_FLYOUT_SECTION_HEADER,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_SELECTOR,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_BUTTON,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_MODAL_TITLE,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_MODAL_SELECT,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_MODAL_SELECT_OPTION,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_MODAL_SAVE_BTN,
+  HOST_DETAILS_FLYOUT_ASSET_CRITICALITY_LEVEL,
+} from '../../../screens/asset_criticality/flyouts';
+import { deleteAlertsAndRules } from '../../../tasks/api_calls/common';
+import {
+  expandFirstAlertHostFlyout,
+  toggleAssetCriticalityAccordion,
+  toggleAssetCriticalityModal,
+} from '../../../tasks/asset_criticality/common';
+import { createRule } from '../../../tasks/api_calls/rules';
+import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
+import { login } from '../../../tasks/login';
+import { visit } from '../../../tasks/navigation';
+import { ALERTS_URL } from '../../../urls/navigation';
+
+describe(
+  'Host details flyout',
+  {
+    tags: ['@ess', '@serverless'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'entityAnalyticsAssetCriticalityEnabled',
+          ])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    const rule = { ...getNewRule(), investigation_fields: { field_names: ['host.os.name'] } };
+
+    beforeEach(() => {
+      deleteAlertsAndRules();
+      login();
+      createRule(rule);
+      visit(ALERTS_URL);
+      waitForAlertsToPopulate();
+    });
+
+    describe('Host flyout', () => {
+      beforeEach(() => {
+        expandFirstAlertHostFlyout();
```

Comment:
NIT: Why not moving this to the above beforeEach?

Reply:
ah, good one! Originally it was because there were other flyouts to test, but that got scrapped. I should move it yes, thanks!